cstar_ocean/cstar_ocean/cstar_environment.py

```diff
@@ -0,0 +1,144 @@
+import os
+import sys
+import platform
+import importlib.util
+from typing import Optional
+
+spec = importlib.util.find_spec(__name__.split(".")[0])
+if spec is not None:
+    if isinstance(spec.submodule_search_locations, list):
+        _CSTAR_ROOT: str = spec.submodule_search_locations[0]
+else:
+    raise ImportError
```

Comment:
```suggestion
    raise ImportError("some explanatory message")
```
if you don't do this then you haven't actually instantiated the error class

Reply:
Fixed in [3e12e874b08df27f7fb148c7d9492c2af52d5859](https://github.com/CWorthy-ocean/C-Star/commit/3e12e874b08df27f7fb148c7d9492c2af52d5859)